source/_components/media_player.openhome.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: "Openhome"
+description: "Instructions how to integrate Openhome into Home Assistant."
+date: 2017-02-21 22:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo:
+ha_category: Media Player
+featured: true
+ha_release: 
```

Comment:
It's easiest to just include the next release for now (0.39) here, so that if we decide to pull it in everything is ready to go. If you miss the release window you can update it.